build.sbt

```diff
@@ -48,16 +48,16 @@ scalacOptions ++= Seq("-unchecked", "-deprecation")
 
 // Code settings
 
-val akkaV = "2.4.8"
+val akkaV = "2.5.0"
 
 libraryDependencies ++= Seq(
   "com.typesafe.akka"         %% "akka-actor"                 % akkaV,
   "com.typesafe.akka"         %% "akka-testkit"               % akkaV,
-  "org.scalatest"             %% "scalatest"                  % "3.0.0-M15" % "test",
-  "org.scalacheck"            %% "scalacheck"                 % "1.12.5" % "test",    // Property-based testing
-  "org.iq80.leveldb"          %  "leveldb"                    % "0.7",                // For LevelDB journal
+  "org.scalatest"             %% "scalatest"                  % "3.0.1"  % "test",
+  "org.scalacheck"            %% "scalacheck"                 % "1.13.4" % "test",    // Property-based testing
+  "org.iq80.leveldb"          %  "leveldb"                    % "0.9",                // For LevelDB journal
```

Comment:
Hmmm....let's remove it. It's a different storage backend than we actually use, and I'm almost positive everything will work just fine without it.